MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -931,6 +976,14 @@ private void PatchName(string nameOrNull)
 			}
 		}
 
+		private void PatchOwner(Guid? ownerOrNull)
```

Comment:
If becoming non-owner, make rigid body kinematic? Maybe that happens elsewhere.

Reply:
Well, it should be handled in physics bridge...
However, whole logic around ownership transfer is a big todo.